codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2ConnectionHandlerBuilder.java

```diff
@@ -399,6 +400,24 @@ protected boolean isAutoAckSettingsFrame() {
         return autoAckSettingsFrame;
     }
 
+    /**
+     * Determine if PING frame should automatically be acknowledged or not.
```

Comment:
Was going to suggest to mention here that the default is `true` but I guess the doc is consistent with `autoAckSettingsFrame`.